pom.xml

```diff
@@ -0,0 +1,314 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    
+    <groupId>fish.payara.microprofile</groupId>
+    <artifactId>tck-suite-parent</artifactId>
+    <version>1.0-SNAPSHOT</version>
+    <packaging>pom</packaging>
+
+    <name>MicroProfile TCK Suite Parent</name>
+    <description>This pom describes how to run all the required MicroProfile TCKs for a release.</description>
+    
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+
+        <!-- Java Version Dependencies -->
+        <maven.compiler.source>1.8</maven.compiler.source>
+        <maven.compiler.target>1.8</maven.compiler.target>
+        
+        <!-- Arquillian Dependencies -->
+        <version.arquillian>1.1.14.Final</version.arquillian>
+        <payara.arquillian.container.version>1.0.Beta3</payara.arquillian.container.version>
+        
+        <!-- Test Dependencies -->
+        <failsafe.version>2.22.1</failsafe.version>
+
+        <!-- Embedded Dependencies -->
+        <json.version>1.1.2</json.version>
+        <tyrus.version>1.13.1</tyrus.version>
+        <jersey.version>2.26</jersey.version>
+
+        <!-- Payara Dependencies -->
+        <payara.version>5.184</payara.version>
+    </properties>
+
+    <modules>
+        <module>MicroProfile-Fault-Tolerance/tck-runner</module>
+        <module>MicroProfile-JWT-Auth/</module>
+        <module>MicroProfile-OpenTracing/tck-runner</module>
+        <module>MicroProfile-Metrics/tck-runner</module>
+        <module>MicroProfile-Rest-Client/tck-runner</module>
+        <module>MicroProfile-Health/</module>
+        <module>MicroProfile-Config/tck-runner</module>
+        <module>MicroProfile-OpenAPI/tck-runner</module>
+    </modules>
+
+    <dependencies>
+        <!-- Arquillian Dependencies -->
+        <dependency>
+            <groupId>org.jboss.arquillian.container</groupId>
+            <artifactId>arquillian-container-test-spi</artifactId>
```

Comment:
Where are the versions for the dependencies? - Perhaps a dependency management section is needed.

Reply:
They're specified by the arquillian bom further up!